Project/DataAccess/Context/UnitOfWork.cs

```diff
@@ -2,12 +2,33 @@
 
 namespace DataAccess.Context
 {
-    public sealed class UnitOfWork : System.IDisposable
+    public sealed class UnitOfWork
     {
         // FIELDS
-        private bool disposedValue = false; // To detect redundant calls
         private AppContext context;
+        private static UnitOfWork instance;

```

Comment:
Do you need this static field?

Reply:
For sure. Thet is where you would store an instance of Singleton value.

Reply:
Ok